invenio_app_rdm/communities_ui/views/ui.py

```diff
@@ -56,6 +57,27 @@ def create_ui_blueprint(app):
         view_func=communities_detail,
     )
 
+    def communities_kwargs(css_class, permissions):
+        kwargs = {}
+        kwargs["class"] = css_class
+        kwargs["permissions"] = permissions
+        return kwargs
+
+    @blueprint.before_app_first_request
+    def register_menus():
+        """Register community menu items."""
+        communities = current_menu.submenu('communities')
+        communities.submenu('search').register(
+            'invenio_app_rdm_communities.communities_detail',
+            text=_('Search'),
+            order=1,
+            expected_args=["pid_value"],
+            **communities_kwargs(
```

Comment:
I think for simplicity you can inject the kwargs directly here without using the `communities_kwargs` function. If you prefer you can do 
```
**dict(class="search icon", permissions=True)
```